src/handlers/GraphQLHandler.ts

```diff
@@ -162,30 +162,37 @@ export class GraphQLHandler<
       return false
     }
 
-    if (!parsedResult.operationName) {
```

Comment:
I believe that the actual fix is simply checking that we're not in the `graphql.operation` handler:

```js
if (!parsedResult.operationName && this.info.operationType !== 'all') {
```